llvm/include/llvm/SYCLLowerIR/DeviceConfigFile.td

```diff
@@ -160,9 +160,34 @@ def : TargetInfo<"x86_64", [], [], "", "", 1>;
 
 // TODO: The aspects listed for the intel_gpu targets right now are incomplete; 
 // only the fp16/fp64/atomic64 aspects are listed.
-def : TargetInfo<"intel_gpu_cfl", [AspectFp16, AspectFp64, AspectAtomic64], [8, 16, 32]>;
-def : TargetInfo<"intel_gpu_tgllp", [AspectFp16, AspectAtomic64], [8, 16, 32]>;
-def : TargetInfo<"intel_gpu_pvc", [AspectFp16, AspectFp64, AspectAtomic64], [16, 32]>;
+defvar Fp16Fp64Atomic64 = [AspectFp16, AspectFp64, AspectAtomic64];
+defvar Fp16Atomic64 = [AspectFp16, AspectAtomic64];
+defvar Sg8_16_32 = [8, 16, 32];
+defvar Sg16_32 = [16, 32];
+defvar IntelBaseAspects = [AspectExt_intel_esimd];
+class IntelTarget<string Name, list<Aspect> Aspects, list<int> subGroupSizesList>
+: TargetInfo<Name, IntelBaseAspects # Aspects, subGroupSizesList>;
+def : IntelTarget<"intel_gpu_pvc", Fp16Fp64Atomic64, Sg16_32>;
```

Comment:
Please also update comment [here ](https://github.com/intel/llvm/blob/1f3f02bd0bc75af9c7d47dbf05bde4ef26279d6b/sycl/include/sycl/ext/oneapi/experimental/device_architecture.hpp#L28), instead when one will add a new architecture, `llvm/include/llvm/SYCLLowerIR/DeviceConfigFile.td` will not be updated

Reply:
[18ea9a9](https://github.com/intel/llvm/pull/14188/commits/18ea9a9dd30c1aed0971fc43415f9005a01d6d31)